### DISCUSSION TITLE
Version  update

### DISCUSSION THREAD

**Discussion by deechella** [argoproj/argo-cd#8406] [General]
can we update directly the argocd  version 2.2.4 from 2.0.0-rc1 using the helm charts

**Comment by ishitasequeira** [comment]
You can check the version upgrade guides below to see if anything affects your workflow.

https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.0-2.1/
https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.1-2.2/
